scripts/flow/common.js

```diff
@@ -0,0 +1,69 @@
+import {weaponEffects} from "../weaponEffects.js";
```

Comment:
Moved from existing files to a common location for both v10 and v11 paths to use.